flow/embedded_views.h

```diff
@@ -18,6 +18,12 @@ class EmbeddedViewParams {
  public:
   SkPoint offsetPixels;
   SkSize sizePoints;
+
+  friend bool operator==(const EmbeddedViewParams& lhs,
```

Comment:
+ @jason-simmons  for C++ style in this part (I've never added a C++ comparison operator in the engine codebase, not sure what to look for)

Reply:
is `friend` necessary here?

Reply:
Done